src/vector/platform/ElectronPlatform.tsx

```diff
@@ -360,6 +360,10 @@ export default class ElectronPlatform extends VectorBasePlatform {
         electron.send('loudNotification');
     }
 
+    needsUrlTooltips(): boolean {
```

Comment:
```suggestion
    public needsUrlTooltips(): boolean {
```

bonus points for fixing all the other functions here: we require explicit member access (but the linter isn't intelligent enough to know that yet)

Reply:
👍 Looks like there were only two other methods that had an explicit access level in `BasePlatform` but not here.